core/src/com/fourinachamber/fortyfive/screen/screenBuilder/FromOnjScreenBuilder.kt

```diff
@@ -126,7 +130,8 @@ class ScreenBuilder(val screenName: String, val onj: OnjObject) : ResourceBorrow
         }
     }
 
-    fun generateFromTemplate(name: String, data: Map<String, Any?>, parent: FlexBox?, screen: OnjScreen): Actor? {
+    override fun generateFromTemplate(name: String, data: Map<String, Any?>, parent: Group?, screen: OnjScreen): Actor? {
+        if (parent !is CustomFlexBox) throw RuntimeException("FromOnjScreenBuilder can only parent templates to FlexBoxes")
```

Comment:
this has to break somewhere, if you want to add something to a cardActor or Image for example